week-1-ts-ecosystem/w01e01-typescript-basics/src/main.ts

```diff
@@ -6,25 +6,11 @@
  *
  * */
 
-const user: any = {
-  name: 'Andy',
-  age: 30,
-  email: 'andy@mail-me-tommorow.com',
-  address: {
-    street: 'Strange Alley',
-    no: 23,
-  },
-}
+import { user } from "./data/userData";
+import hasAddress from "./helpers/hasAddress";
+import hasGivenAge from "./helpers/hasGivenAge";
 
-function hasAddress(user: any): any {
-  return Boolean(user.address)
-}
+const isAdult = hasGivenAge(18);
```

Comment:
To `isAdult` to też w pewnym sensie `helper` / (validator) - powinien mieć swoje miejsce :) idąc konsekwentnie z Twoim pomysłem 😺